qaframework-bdd-tests/src/test/resources/features/refapp-2.x/03-trackPatient/editDemographicsAndContactInfo.feature

```diff
@@ -1,21 +1,27 @@
 Feature: Edit Demographics and contact info from Patient Dashboard
 
-  Background:
-    Given a user clicks on Edit Registration Information link from Patient dashboard
-    Then the system loads Registration Summary Page
-
   @selenium
   @patientDemographics
   Scenario: Edit Patient Demographics and Contact Info
-    # User story: Edit Patient Demographics
-    When a user clicks on Edit link from Registration Summary Page
-    Then the system loads edit demographics section
+    Given a user clicks on the find patient app on the home page
+    Then the system loads the find patient page
+    And a user searches for the patient to edit
+    And a user clicks on the first patient
+    And a user clicks on Edit Registration Information link from Patient dashboard
+    Then the system loads Registration Summary Page
     And a user edits demographics
-    And a user clicks on the confirm button
```

Comment:
any reason why the user story is being changed?

Reply:
Am trying as much as possible to follow the contactInfoTest and EditDemographicsTest selenium tests. Changing that user story is part of it.

Reply:
According to this commit. Hopefully @kdaud  will find it more helpful concerning this comment [here](https://issues.openmrs.org/projects/RATEST/issues/RATEST-308?filter=reportedbyme)

Reply:
@mherman22 the user story am referring to is 

```js
And a user clicks on the confirm button

```
changed to 

```js
And a user confirms the changes
```

The earlier phrasing of the user step is more clear.